### DISCUSSION TITLE
Login vs Registration through OIDC

### DISCUSSION THREAD

**Discussion by johanforssell** [ory/kratos#1252] [Q&A] [answered]
I'm a bit lazy here, not going through the code https://github.com/ory/kratos-selfservice-ui-node first, but perhaps someone can tell me straight away:

Is there a difference between logging in and registering when you go through OIDC? Looking at the selfservice-ui, there does not seem to be one.

No matter if I click the "Continue with _SERVICE_" on the login or the registration page I'm getting logged in.

### References
- [Registration through OIDC](https://www.ory.sh/kratos/docs/self-service/flows/user-registration#registration-with-google-facebook-github--openid-connect--oauth-20)
- [Login through OIDC](https://www.ory.sh/kratos/docs/self-service/flows/user-login#login-with-google-facebook-github--openid-connect--oauth-20)

**Comment by aeneasr** [comment]
https://www.ory.sh/kratos/docs/self-service/hooks/#registration

**Reply by johanforssell** [reply]
Thank you. But I was thinking about the Kratos identity getting auto-created. Not that the user is auto-logged in.

**Comment by johanforssell** [comment, marked as answer]
I was very bad at reading the documentation - there's actually a blurb about my exact question already.

> ![image](https://user-images.githubusercontent.com/684381/115673141-948c5980-a34c-11eb-8646-d2e77622551e.png)

It's listed under [Self Service (End-User) ↣ User Login ↣ Login with google, Facebook, Github, ..., OpenID Connect / OAuth 2.0](https://www.ory.sh/kratos/docs/self-service/flows/user-login#login-with-google-facebook-github--openid-connect--oauth-20-1)
